lawnchair/src/app/lawnchair/qsb/providers/Yandex.kt

```diff
@@ -0,0 +1,22 @@
+package app.lawnchair.qsb.providers
+
+import android.content.Intent
+import app.lawnchair.qsb.ThemingMethod
+import com.android.launcher3.R
+
+data object Yandex : QsbSearchProvider(
+    id = "yandex",
+    name = R.string.search_provider_yandex,
+    icon = R.drawable.ic_yandex,
+    themingMethod = ThemingMethod.THEME_BY_LAYER_ID,
```

Comment:
Please use the following instead, since the svg icon doesn't use custom layers.

```suggestion
    themingMethod = ThemingMethod.TINT,
```

Reply:
I tried using ThemingMethod.TINT, but with it the icon looks just like a solid circle, but ok, I'll try to find a more suitable icon for theming support

Reply:
![Screenshot_20240706-222349](https://github.com/LawnchairLauncher/lawnchair/assets/37577810/7b875bb0-1400-4648-a374-373d933c14b9)